### ISSUE TITLE
Please give me an option to limit the constellations in the "Character" screen

### ISSUE THREAD

**Issue by codemasher** [MadeBaruna/paimon-moe#384]
Hey, it would be nice if i could limit the display of constellations to the standard maximum of C6. 
How about a character counter table under "Wish Counter" instead?

![image](https://user-images.githubusercontent.com/592497/219699747-a219dbe8-5d7a-4ecc-994f-eeb0b6ab6f34.png) ![image](https://user-images.githubusercontent.com/592497/219700112-80585784-401e-4e1c-b87a-b83003da550c.png)

**Comment by jogerj** [comment]
No, you want to see duplicate constellations. This is harmless anyways

**Comment by codemasher** [comment]
> No, you want to see duplicate constellations. This is harmless anyways

What's wrong with having the choice?

**Comment by Ilya0S** [comment]
Maybe a compromise? A button. If you press it, all your C7+ characters become C6 characters.
This way, no need for any hard-to-make automations, but also a freedom of choice.

**Comment by codemasher** [comment]
This is what i proposed. An OPTION to just display C6 as maximum. I don't want to see how many of the same characters i got in that screen. I also suggested an addition to the wish counter screen, a table that shows the total number of characters pulled.
Reading comprehension seems hard sometimes jfc.

**Comment by Ilya0S** [comment]
Yeah, but I meant a button instead of checkbox. So it doesn't check every time `if Const>6 then Const:=6` whenever character constellation gets added but instead, whenever you press that button, it runs `for (i:=0; i<=Characters.length; i++) do { if Characters[i].Const>6 then Characters[i].Const:=6 }`

**Comment by codemasher** [comment]
a) it doesn't matter what or when or how it checks as everything runs on the client side and it's not like it takes 10 minutes to loop through
b) i don't want to press a button every single time. there is a settings tab already, so put it to good use.

**Comment by Ilya0S** [comment]
a) it matters for those who code it
b) I guess the fact that settings tab don't have many settings shows that those who code it don't want to have many settings
I don't say for them to do either a button or nothing, I say "if you don't want to make a setting, make a button at least"

**Comment by codemasher** [comment]
we're talking about this line optionally being `Math.max()`'ed to 6, nothing more https://github.com/MadeBaruna/paimon-moe/blob/77c85c383a9fc32c4c033aae5285d6039987c6cb/src/routes/characters/_characterGridItem.svelte#L56
